src/resource_clients/run.ts

```diff
@@ -138,6 +141,34 @@ export class RunClient extends ResourceClient {
         return cast(parseDateFields(pluckData(response.data)));
     }
 
+    /**
+     * https://docs.apify.com/api/v2#/reference/actor-runs/charge-run/charge-run
+     */
+    async charge(options: RunChargeOptions): Promise<ApifyResponse<Record<string, never>>> {
+        ow(options, ow.object.exactShape({
+            eventName: ow.string,
+            count: ow.optional.number,
+            idempotencyKey: ow.optional.string,
+        }));
+
+        const count = options.count ?? 1;
+        const idempotencyKey = options.idempotencyKey ?? `${this.id}-${options.eventName}-${Date.now()}`;
```

Comment:
Same note here as in the Python client, the idempotency key has to be a unique string. This way, if someone charged for two occurences of the same event in the same millisecond, only one of the charges would go through.

https://github.com/apify/apify-client-python/pull/304#pullrequestreview-2480812542